go18/plan.sh

```diff
@@ -0,0 +1,119 @@
+pkg_name=go18
```

Comment:
So it is appropriate for this to stay as `go` and not become `go18` just yet. We want `core/go` to always resolve to the latest major version of the language. We need to add a new type of package called a "pointer package" to get this behaviour, but for now we just do it by naming convention.